backend/src/v5/middleware/dataConverter/outputs/teamspaces/projects/models/commons/tickets.comments.js

```diff
@@ -0,0 +1,41 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { respond } = require('../../../../../../../utils/responder');
+const { serialiseComment } = require('../../../../../../../schemas/tickets/comments');
+const { templates } = require('../../../../../../../utils/responseCodes');
+
+const Comments = {};
+
+Comments.serialiseComment = (req, res) => {
+	try {
+		respond(req, res, templates.ok, serialiseComment(req.commentData));
+	} catch (err) {
+		respond(req, res, templates.unknown);
```

Comment:
Can we log these errors to the logger? Would be much easier to debug if we have it somewhere